dind/Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM alpine:3.5
+FROM alpine:3.19
 
-ENV DOCKER_VERSION=1.13.1 \
-  DOCKER_COMPOSE_VERSION=1.20.1 \
-  DOCKER_PACKAGES="curl device-mapper mkinitfs bash e2fsprogs e2fsprogs-extra iptables python-dev" \
+ENV DOCKER_VERSION=19.03.9 \
+  DOCKER_COMPOSE_VERSION=1.24.1 \
+  DOCKER_PACKAGES="curl device-mapper mkinitfs bash e2fsprogs e2fsprogs-extra iptables python3-dev" \
```

Comment:
was able to use 3 major versions newer docker runtime and 4 minor versions newer docker-compose (which supports easier logs output)